content/updates/2018-03.md

```diff
@@ -0,0 +1,10 @@
+---
+title: April 2018
+version: 2018.3.0
+date: 2018-04-17
+hidden: true
+image: https://270-97323138-gh.circle-artifacts.com/0/pi-image.img.xz
+---
+
+- Allow setting the starting zone of a robot using a USB drive. This also sets the robot to competition mode.
```

Comment:
Maybe describe as a "competition USB drive" to prevent people trying to do this themselves and failing?

Reply:
Will add. However it's undocumented, and we're already checking if a `main.py` exists on the drive, so we _should_ be fine

Reply:
I realise that's why it will fail, my concern is that that failure will confuse the competitors.